### DISCUSSION TITLE
Additional devtools layouts

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#4607] [Ideas]
At the moment we have the one vscode style ide layout, but it would also be nice to have a toolbox layout where the toolbox is below the video or to the left of the video

### Left

<img width="1499" alt="Screen Shot 2021-11-20 at 6 38 15 PM" src="https://user-images.githubusercontent.com/254562/142747235-587649df-baa8-4727-a686-2cb395c9300a.png">

**Comment by jonbell-lot23** [comment]
This is something we've discussed for a long time and I know we're eager to address.

The drawback of the visual as shown here is that the editor has gone away. So any design we work through needs to explain how to show/hide the editor, demonstrate what the default state looks like, and understand what actions auto-deploy the editor. It's worth stressing that our number one ah-ha moment (print statements) requires the editor.

Here's the design point of view as of March 7, 2022: https://www.loom.com/share/879984e1fec1474793d2d38c703e1d2e

**Reply by jonbell-lot23** [reply]
We've been iterating and here's a Notion page for tracking that: https://www.notion.so/replayio/Layout-v2-ced633258f234a188cc50095ea83e984

**Comment by jonbell-lot23** [comment]
This has now shipped! As I mentioned above, there are nuances we're still working through:

* How to show/hide the editor: addressed
* What the default state looks like: addressed
* Understand what actions auto-deploy the editor: right now it's kinda broken and we're working through that
